sources/packages/web/src/components/common/AwardTable.vue

```diff
@@ -55,13 +59,26 @@ export default defineComponent({
       if (awardValue === null) {
         return "-";
       }
+      if (awardValue) {
```

Comment:
If the award value is `0` the expectation is that it must be displayed as `$0.00`.

But `if(0) will be false` and that makes the value returned as `(Not eligible)` when the value is `0`. 

Changing the code in following way will fix it. 

```ts
// If the award is defined but no values are present it means that a receipt value is missing.
      if (awardValue === null) {
        return "-";
      }

      // If the award in not defined at all it means that the award is not eligible and it was not
      // part of the disbursement calculations output.
      if (awardValue === undefined) {
        return "(Not eligible)";
      }

      return (
        "$" + awardValue.toLocaleString("en-CA", { minimumFractionDigits: 2 })
      );
```